pom.xml

```diff
@@ -3,17 +3,17 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.48</version>
+    <version>4.47</version>
+    <relativePath />
   </parent>
 
   <properties>
-    <jenkins.version>2.60.3</jenkins.version>
-    <configuration-as-code.version>1.27</configuration-as-code.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.346.1</jenkins.version>
+    <spotbugs.failOnError>false</spotbugs.failOnError>
   </properties>
 
   <artifactId>oic-auth</artifactId>
-  <version>1.9-SNAPSHOT</version>
+  <version>2.0.0</version>
```

Comment:
I think this will break, version should be SNAPSHOT until you do a release, which will update the version number